.github/workflows/release.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
 
   release:
     needs: check
-    runs-on: ubuntu-24.04
+    runs-on: ${{ matrix.runs-on }}
```

Comment:
@chanchiwai-ray I think this will fail because the is no `matrix.runs-on` defined
:thinking: 

The soleng repo seems fine...not sure what is happening here.

Feel free to resolve if the comment is not true

Reply:
Looks like you're right. There's a bug in automation where it won't make a change to this existing branch if the main branch already matches automation.